okhttp/src/main/java/okhttp3/ResponseBody.java

```diff
@@ -103,7 +103,10 @@ public final Reader charStream() {
   /**
    * Returns the response as a string decoded with the charset of the Content-Type header. If that
    * header is either absent or lacks a charset, this will attempt to decode the response body as
-   * UTF-8.
+   * UTF-8. Closes {@link ResponseBody} automatically.
+   *
+   * <p>Warning: this method loads whole response body into memory and may cause
```

Comment:
Very sorry, but not sure I understand what exactly do you want to qualify… 

// Asked my US friend-coworker to help because I thought that it was lack of my understanding (English), but he is also not sure what needs to be qualified.